skyfield/documentation/positions.rst

```diff
@@ -39,7 +39,8 @@ together with all of the attributes and methods that they support:
 
     Apparent position only
      │
-     └── `altaz(…) <api.html#Position.altaz>`_            →   alt, az, distance
+     └── `altaz(…) <api.html#Position.altaz>`_                 →   alt, az, distance
+     └── `over_location(…) <api.html#Apparent.over_location>`_ →   topos
```

Comment:
Would like to cast +1 for `sub-point` instead of `over_location`. See https://celestrak.com/columns/v02n03/.